sunbeamlib/initialize.py

```diff
@@ -42,8 +46,11 @@ def main():
     else:
         args.conda_fp = args.conda_fp.absolute()
 
-    config = create_blank_config(
-        args.conda_fp, args.project_fp, template=args.server)
-    
+    if args.template is None:
+        config = create_blank_config(
+            args.conda_fp, args.project_fp, template=args.server)
+    else:
+        config = args.template.read() 
     sys.stdout.write(config)
 
+main()
```

Comment:
This isn't necessary (setup.py sets it so that the command line script calls `main()` automatically) but if it's here it should be the canonical form:
```python
if __name__ == "__main__":
  main()
```
This lets the file get imported without executing the main() function (it only executes if it's the main process)